### DISCUSSION TITLE
Character (like *) to display all objects that contain a certain string

### DISCUSSION THREAD

**Discussion by ajtonyb** [postalserver/postal#2432] [Help with using Postal]
Hello! I am looking for the character with which i can filter all the outgoing messages that contain "@example.com" domain or any other substring. I have tried, with all the usual characters, but none worked. Do you have any suggestions, or is it even possible on website-level?

**Comment by sinnbeck** [comment]
That does not seem to be possible sadly. It uses this to do the sql query to get the messages
https://github.com/postalserver/postal/blob/5222263484f2a5889653859db6aeb7438f116357/app/controllers/messages_controller.rb#L171
```
options[:where][:rcpt_to] = qs[:to] if qs[:to]
```
I believe you would need something like this to use a "wildcard"
```
options[:where][:rcpt_to] = "%#{qs[:to]}%" if qs[:to]
```
(I am not a ruby developer but can mostly figure out what it does :)

**Reply by ajtonyb** [reply]
Sad to hear, but thanks anyway. It would be a very useful feature, hopefully gets implemented someday.

**Reply by sinnbeck** [reply]
Yeah I agree that indeed would be nice :)

**Comment by willpower232** [comment]
Postal isn't designed to hold on to messages long term so doesn't have a great search form for messages. Someone proposed adding subject line to the searched fields but didn't finish it #1594 

If you need to interrogate the messages you send long term, they need to be stored at the source and not by Postal.

Whilst the answer may be a wildcard, I don't know enough Ruby to know if that introduces an exploitable angle.
